src/AtomBeautify.ts

```diff
@@ -85,13 +93,23 @@ export class AtomBeautify {
         text = editor.getText();
       }
       let fileExtension: string = null;
-      if (editor.getPath()) {
-        fileExtension = path.extname(editor.getPath()).substr(1);
+      const editorPath = editor.getPath();
+      if (editorPath) {
+        fileExtension = path.extname(editorPath).substr(1);
       }
+      const langs: Language[] = this.unibeautify.findLanguages({
+        atomGrammar: grammarName,
+        extension: fileExtension
+      });
+      const language = langs.length > 0 ? langs[0] : null;
+      const configSettings = this.getConfigFromSettings(language.name);
+      const beautifySettings = await this.unibeautifyConfiguration(configSettings, editorPath, language.name);
       return this.unibeautify.beautify({
         fileExtension,
         atomGrammar: grammarName,
-        options: {},
+        options: {
+          [language.name]: beautifySettings
+        },
```

Comment:
Should be passing in the configs for all languages. A single `beautify` could end up beautifying multiple languages. Consider `PHP` as root language and having embedded `HTML`.

Reply:
I agree.  At the same time that feature warrants more discussion.

Reply:
Here's what I did for VSCode:
- https://github.com/Unibeautify/vscode/blob/master/src/EditProvider.ts#L102-L115
- https://github.com/Unibeautify/vscode/blob/master/src/EditProvider.ts#L60-L68